references/product-recommendations-api/pipeline.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo [INFO] Pipeline for product-recommendations-api in project: "$PROJECT_ID"
+
+# Set environment variables and default project for gcloud commands 
+source set_env_variables_my_apigeex.sh
```

Comment:
use the Apigee X variables for the nightly.

APIGEE_X_ORG
APIGEE_X_ENV
APIGEE_X_HOSTNAME

Reply:
OK, updated.